src/usr/local/www/services_dhcp.php

```diff
@@ -1636,7 +1636,14 @@ function build_pooltable() {
 ?>
 
 <div class="panel panel-default">
-	<div class="panel-heading"><h2 class="panel-title"><?=gettext("DHCP Static Mappings for this Interface")?></h2></div>
+<?php
+	if (is_array($a_maps) && (count($a_maps) > 0)) {
+		$title = gettext("DHCP Static Mappings for this Interface (total: " . count($a_maps) . ")");
```

Comment:
You shouldn't use variables in `gettext()` strings. It should be run through `sprintf()` to replace the value. Something like this:

    $title = sprintf(gettext('DHCP Static Mappings for this Interface (total: %d)'), count($a_maps));

Reply:
ok, fixed